<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de Pilotos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../incon_Main.png</normaloff>../incon_Main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(175, 175, 175);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_Main" rowstretch="0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox_TipoDeOrdenacao">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <item>
        <property name="text">
         <string>CODIGO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOME</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EQUIPE e NOME</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PAIS e NOME</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MOTOR, EQUIPE e NOME</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PONTUAÇÃO</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_Ordenar">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>ORDENAR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Limpar">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>LIMPAR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_CarregarArquivo">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>CARREGAR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QTableWidget" name="tableWidget_Mostrar">
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>CODIGO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PILOTO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PAÍS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IDADE</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>EQUIPE</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MOTOR</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PONTOS</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
